### PR TITLE
machine_configuration: Update readme and remove deprecated MCO

### PR DESCRIPTION
**Pull request by hershpa**
Update OCP version number to 4.13
Update readme section for instructions to deploy and verify MCO to disable incompatible drivers. Remove old MCO to blacklist ast as it is no longer needed for RHOCP 4.13. Signed-off-by: Hersh Pathak hersh.pathak@intel.com

### CODE REVIEW

File: machine_configuration/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ If the configuration cannot be set as the default setting, we recommend using so
 Any contribution in this area is welcome. 
 
 # Prerequisites 
-- Provisioned RHOCP 4.12 cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster).
+- Provisioned RHOCP 4.13 cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster).
```
Comment (uMartinXu) [inline machine_configuration/README.md:19]: According to @vbedida79's suggestion, let us remove the OCP version here, and just say provisioned RHOCP cluster

File: machine_configuration/README.md [hunk 2]
```diff
@@ -70,22 +70,23 @@ intel-dgpu   rendered-intel-dgpu-58fb5f4d72fe6041abb066880e112acd   True      Fa
 ```
 Ensure `intel-dgpu` MachineConfigPool is present.
 
-# Disable conflicting driver
-Run the command shown below to disable the loading of a potential conflicting driver, such as `ast` driver.
+# Disable incompatible in-tree drivers
```
Comment (uMartinXu) [inline machine_configuration/README.md:73]: Suggest to just say
# Disable in-tree drivers

File: machine_configuration/README.md [hunk 3]
```diff
@@ -70,22 +70,23 @@ intel-dgpu   rendered-intel-dgpu-58fb5f4d72fe6041abb066880e112acd   True      Fa
...
+Run the command shown below to disable the loading of incompatible in-tree drivers, including `i915` and `intel_vsec` driver.
```
Comment (uMartinXu) [inline machine_configuration/README.md:74]: to disable the loading of in-tree drivers 'i915' and 'intel_vsec'

File: machine_configuration/README.md [hunk 4]
```diff
@@ -70,22 +70,23 @@ intel-dgpu   rendered-intel-dgpu-58fb5f4d72fe6041abb066880e112acd   True      Fa
...
 
-**Note**: The `i915` driver depends on a ported `drm` module. Some other drivers, such as ast that depends on in-tree drm module might have a compatibility issue. The known issue will be resolved on i915 driver for RHEL `9.x`, which will be used for RHOCP `4.13`. 
+**Note**: The in-tree i915 and intel_vsec driver do not provide support for the Intel Data Center GPU cards. As a result, it is required as a prerequisite to disable loading of the in-tree drivers.
```
Comment (uMartinXu) [inline machine_configuration/README.md:76]: suggest to remove the note.

File: machine_configuration/README.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ If the configuration cannot be set as the default setting, we recommend using so
 Any contribution in this area is welcome. 
 
 # Prerequisites 
-- Provisioned RHOCP 4.12 cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster).
```
[the remark this answers is not in the document]
Comment (hershpa) [reply machine_configuration/README.md:19]: Sounds good

File: machine_configuration/README.md (hunk 2 above)
Comment (hershpa) [reply machine_configuration/README.md:73]: Got it

File: machine_configuration/README.md (hunk 1 above)
Comment (hershpa) [reply machine_configuration/README.md:19]: Sounds good

**Pull request closed without merging**